packages/app-builder-lib/templates/nsis/include/allowOnlyOneInstallerInstance.nsh

```diff
@@ -30,12 +30,12 @@
 !macroend
 
 !macro CHECK_APP_RUNNING
-  Var /GLOBAL CmdPath
-  Var /GLOBAL FindPath
-  Var /GLOBAL PowerShellPath
   !ifmacrodef customCheckAppRunning
     !insertmacro customCheckAppRunning
   !else
+    Var /GLOBAL CmdPath
+    Var /GLOBAL FindPath
+    Var /GLOBAL PowerShellPath
```

Comment:
I'm looking further at the code and it seems to assume that `CmdPath` `FindPath` and `PowerShellPath` must exist for further functionality to succeed. I think moving the `Var`s here may actually cause a downstream bug of a var not being defined, no?
I'm looking at
`IS_POWERSHELL_AVAILABLE`
https://github.com/electron-userland/electron-builder/blob/44b28997f15314730d1bb69303a47dc26f7950d1/packages/app-builder-lib/templates/nsis/include/allowOnlyOneInstallerInstance.nsh#L50-L57
`FIND_PROCESS`
https://github.com/electron-userland/electron-builder/blob/44b28997f15314730d1bb69303a47dc26f7950d1/packages/app-builder-lib/templates/nsis/include/allowOnlyOneInstallerInstance.nsh#L66-L70

Reply:
Are those functions/macros not only called when not defining `customCheckAppRunning`?

I guess `StrCpy` could be moved out instead?

Reply:
Yeah, I'm thinking move it out instead, and then the `customCheckAppRunning` can override/overwrite those `Vars` if necessary?